### PR TITLE
Added multi-factor unlock

### PR DESCRIPTION
**Pull request by joinimran**
Added multifactor unlock feature update using Passport for work CSP.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/4700

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/identity-protection/hello-for-business/feature-multifactor-unlock.md [hunk 1]
```diff
@@ -332,6 +332,7 @@ The Group Policy object contains the policy settings needed to trigger Windows H
 > * PIN **must** be in at least one of the groups
 > * Trusted signals **must** be combined with another credential provider
 > * You cannot use the same unlock factor to satisfy both categories. Therefore, if you include any credential provider in both categories, it means it can satisfy either category, but not both.
+> * The multifactor unlock feature is also supported via the Passport for Work CSP, see [Passport For Work CSP](https://docs.microsoft.com/windows/client-management/mdm/passportforwork-csp).
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/hello-for-business/feature-multifactor-unlock.md]:
```suggestion
> * The multifactor unlock feature is also supported via the Passport for Work CSP. See [Passport For Work CSP](https://docs.microsoft.com/windows/client-management/mdm/passportforwork-csp) for more information.
```

**Comment by joinimran** [comment]
Suggestions applied.

**Comment by mypil** [comment]
@mapalko @Dansimp - PR has been copy edited and is ready for final review. This is based on issue #4700.

Thank you.

**Comment by illfated** [comment]
The rest of the page needs some MarkDown codestyle improvements, as well as XML codestyle fine tuning and marking the XML code blocks as **```xml** (but that is outside the scope of this PR, so I will get back to this page at a later time (after the merge of this PR) and see what I can do to improve it.

**Comment by mypil** [comment]
@Dansimp - Can you possibly merge this PR? This has been approved by the author @mapalko 

Thank you.

**Pull request merged**